tasks/managed/create-advisory/README.md

```diff
@@ -15,6 +15,7 @@ Only all `redhat-pending` or all `redhat-prod` repositories may be specified in
 | dataPath                 | Path to data JSON in the data workspace                                                                                    | No        | -                       |
 | resultsDirPath           | Path to results directory in the data workspace                                                                            | No        | -                       |
 | request                  | Type of request to be created                                                                                              | Yes       | create-advisory         |
+| contentType              | The contentType of the release artifact. One of [image|generic]                                                            | Yes       | image                   |
```

Comment:
You're adding two more new parameters, so those should be added here as well: cgwHostname and cgwSecret